riemann/fetch_riemann_dash_config.yml

```diff
@@ -0,0 +1,9 @@
+---
+- hosts:
+    - riemann
+  tasks:
+    - name: download
+      become: yes
+      fetch:
+        src: /etc/riemann-dash/config.json
+        dest: .
```

Comment:
Is `.` the right destination path? The file is copied from `roles/riemann-dash/files/etc/riemann-dash/config.json` later.